src/util-host-os-info.c

```diff
@@ -344,6 +344,11 @@ void SCHInfoLoadFromConfig(void)
             int is_ipv4 = 1;
             if (index(host->val, ':') != NULL)
                 is_ipv4 = 0;
+            if ((! strcmp(policy->name, "bsd-right")) ||
+                (! strcmp(policy->name, "old-solaris"))) {
+                SCLogWarning(SC_ERR_UNIMPLEMENTED, "bsd-right and old-solaris are unimplemented"
+                             " and default to bsd (resp. solaris)");
```

Comment:
The message is a bit confusing, esp for old-solaris. I think 2 different messages, one for each setting, would be more clear here.